src/logic/playercommand.cc

```diff
@@ -2056,7 +2056,7 @@ void CmdToggleMuteMessages::read(FileRead& fr, EditorGameBase& egbase, MapObject
 		if (packet_version == kCurrentPacketVersionCmdToggleMuteMessages) {
 			PlayerCommand::read(fr, egbase, mol);
 			building_ = fr.unsigned_32();
-			all_ = fr.unsigned_8() ? 1 : 0;
+			all_ = fr.unsigned_8() == 1;
```

Comment:
```suggestion
			all_ = fr.unsigned_8();
```
Simplify for consistency. Same below.